.circleci/config.yml

```diff
@@ -250,6 +266,29 @@ commands:
           command: |
             psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
             psql -c 'create database webservice_test with owner = dockstore;' -U postgres
+  old_python: # Need Python 3.6 for old Dockstore 1.7 requirements.txt
```

Comment:
I established at release post-mortem that I don't know regression tests that well, but I thought the idea of the regression tests was to test the previous CLI version with the current web service. So shouldn't this be 1.9?

Reply:
1.7, 1.8, & 1.9 all use the same pip requirements